crux-metrics/src/crux/metrics/query.clj

```diff
@@ -1,20 +1,37 @@
 (ns ^:no-doc crux.metrics.query
   (:require [crux.bus :as bus]
-            [crux.metrics.dropwizard :as dropwizard]))
+            [crux.node :as node]
+            [crux.query :as q]
+            [crux.metrics.dropwizard :as dropwizard])
+  (:import java.util.Date))
 
 (defn assign-listeners
   [registry {:crux.node/keys [bus]}]
   (let [!timer-store (atom {})
         query-timer (dropwizard/timer registry ["query" "timer"])]
     (bus/listen bus {:crux/event-types #{:crux.query/submitted-query}}
-              (fn [event]
-                (swap! !timer-store assoc (:crux.query/query-id event) (dropwizard/start query-timer))))
+                (fn [event]
+                  (swap! !timer-store assoc (:crux.query/query-id event) (dropwizard/start query-timer))))
 
     (bus/listen bus {:crux/event-types #{:crux.query/completed-query}}
-              (fn [event]
-                (dropwizard/stop (get @!timer-store (:crux.query/query-id event)))
-                (swap! !timer-store dissoc (:crux.query/query-id event))))
+                (fn [event]
+                  (dropwizard/stop (get @!timer-store (:crux.query/query-id event)))
+                  (swap! !timer-store dissoc (:crux.query/query-id event))))
     {:query-timer query-timer
      :current-query-count (dropwizard/gauge registry
                                             ["query" "currently-running"]
                                             (fn [] (count @!timer-store)))}))
+
+(defn add-slow-query-listeners
```

Comment:
could/should this be included in `assign-listeners`, above?

Reply:
that, or debating whether it has much to do with metrics at all, in hindsight. wdyt?

Reply:
See latest commits - moved it out to the listeners within crux.node and did some refactoring. Unsure if it should, indeed, live there, but considering thats where the current configuration and listeners for it live it made sense.